src/hooks/useDataSource.ts

```diff
@@ -114,8 +115,12 @@ async function importBook(storage: BookStorage | null, rawData: Uint8Array) {
   const rawBook = await importGnucashBook(parsedData);
   await DATASOURCE.sqljsManager.loadDatabase(rawBook);
 
-  // Remove all previous data from the state
-  mutate((key: string) => key.startsWith('/api'), undefined);
```

Comment:
This creates problems as we mutate keys that will not exist anymore and thus are `undefined` when SWR iterates over them